DigitalLearningSolutions.Web/Controllers/PricingController.cs

```diff
@@ -1,17 +1,33 @@
 ﻿namespace DigitalLearningSolutions.Web.Controllers
 {
     using DigitalLearningSolutions.Data.Enums;
+    using DigitalLearningSolutions.Data.Helpers;
     using DigitalLearningSolutions.Web.Attributes;
     using DigitalLearningSolutions.Web.Models.Enums;
     using Microsoft.AspNetCore.Mvc;
+    using Microsoft.Extensions.Configuration;
 
     [SetDlsSubApplication(nameof(DlsSubApplication.Main))]
     [SetSelectedTab(nameof(NavMenuTab.Pricing))]
     public class PricingController : Controller
     {
+        private readonly IConfiguration configuration;
+
+        public PricingController(
+            IConfiguration configuration
+        )
+        {
+            this.configuration = configuration;
+        }
+
         [RedirectDelegateOnlyToLearningPortal]
         public IActionResult Index()
         {
+            if (!configuration.IsPricingPageUsed())

```

Comment:
Can we not use `[FeatureGate(FeatureFlags.PricingPage)]` on the controller like we do with the tracking system pages?

Reply:
Oh nice, didn't know about that. I've added it in, and this makes the tests that I added unnecessary, right?

Reply:
Yeah it does